vpr/src/route/route_net.h

```diff
@@ -116,4 +115,13 @@ void update_rr_base_costs(int fanout);
  * to reflect that these nodes have already been routed to */
 void update_rr_route_inf_from_tree(const RouteTreeNode& rt_node);
 
+/* Convert sink mask to a vector of net pin indices. */
```

Comment:
1. Can explain a bit more: every '1' bit in the mask will turn into the appropriate sink number being stored and returned in the returned vector.
2. Make the comment Doxygen compatible.